core-web/libs/sdk/uve/src/lib/editor/public.ts

```diff
@@ -125,7 +125,7 @@ export function enableBlockEditorInline(
     contentlet: DotCMSBasicContentlet,
     fieldName: string
```

Comment:
try typing this fieldName, instead of string. Test it out, I think it can be disruptive for the user when passing the value, so if it will give trouble to them, leave the `as` this apply to my comment related above

Reply:
The thing with typing, is that we probably will need to make it generic, because sometimes the field is not part of the base contentlet, but a custom property